Automattic/jetpack
# Jetpack CRM - Modern Invoice template (improves CSS and layouts)

Some HTML fixes 9080470-zd-a8c

## Proposed changes:
Fixes some issues with the default PDF template, created a new "modern" template to keep backwards compatibility. Also removed some UI issues where the invoicing pro upsell was huge. 

### Other information:


## Does this pull request change what data or activity we track or use?
N/o

## Testing instructions:
- Switch to this branch
- Check the Invoice List view, the large upsell for invoicing pro is removed
- Create an invoice, add line items and all the relevant details
- Assign some translations with a lower value than the invoice
- In Jetpack CRM settings, choose the new template
- Download the PDF should look like the below

The invoice should look like this (modern compared to default)
[DEFAULT (TRUNK) invoice-1 (64).pdf](https://github.com/user-attachments/files/18054131/invoice-1.64.pdf)
[MODERN (THIS BRANCH) invoice-1 (62).pdf](https://github.com/user-attachments/files/18054110/invoice-1.62.pdf)
